tests/bluetooth/crypto_toolbox/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_ENABLE_CRYPTO_TOOLBOX
+	bool
+	default y
+	select BT_CRYPTO_TOOLBOX
```

Comment:
we are in a dedicated test sample, so why have this as a `select BT_CRYPTO_TOOLBOX` on not directly as a `CONFIG_BT_CRYPTO_TOOLBOX=y` in the prj.conf ?